ftw/simplelayout/browser/resources/Overlay.js

```diff
@@ -18,6 +18,7 @@
         left: "center",
         fixed: false,
         closeOnClick: false,
+        cssclass: "overlay-delete",
```

Comment:
I dont think that `delete` is a good default - shouldn't this be defined where the delete overlay is invoked?